### PR TITLE
Kapil august integration

### PR DESCRIPTION
**Pull request by kapil-ms**

### CODE REVIEW

File: DeviceAdministration/Infrastructure/Repository/DeviceRegistryRepositoryND.cs [hunk 1]
```diff
@@ -233,6 +252,29 @@ public DeviceRegistryRepositoryND(IConfigurationProvider configProvider, IDocDbR
             };
         }
 
+        public async Task<DeviceListQueryResultND> GetDeviceListND(DeviceListQuery query)
+        {
+            List<dynamic> deviceList = await GetAllDevicesAsync();
+
```
Comment (handeadiguzel) [inline DeviceAdministration/Infrastructure/Repository/DeviceRegistryRepositoryND.cs]: can we change this dynamic too?

Comment (kapil-ms) [inline DeviceAdministration/Infrastructure/Repository/DeviceRegistryRepositoryND.cs]: Yes, will do

Comment (kvish) [inline DeviceAdministration/Infrastructure/Repository/DeviceRegistryRepositoryND.cs]: Thing we can convert the deviceList in GetAllDevicesAsync to DeviceND objects and return that list

Comment (kapil-ms) [inline DeviceAdministration/Infrastructure/Repository/DeviceRegistryRepositoryND.cs]: Not that simple. We have to make FilterHelper to use strongly typed objects.

**Pull request merged**